openquake/hazardlib/contexts.py

```diff
@@ -46,7 +46,7 @@
 from openquake.hazardlib.probability_map import ProbabilityMap
 from openquake.hazardlib.geo.surface import PlanarSurface
 
-STD_TYPES = (StdDev.TOTAL, StdDev.INTER_EVENT, StdDev.INTRA_EVENT)
+STD_TYPES = (StdDev.TOTAL, StdDev.INTER_EVENT, StdDev.INTRA_EVENT, StdDev.SINGLE_STATION)
```

Comment:
For 10 years we have been able to work without introducing other kinds of standard deviations, I would keep it that way. Please at the other GMPEs using phi_ss, they do not require to change STD_TYPES.

Reply:
Yes it has been removed.